internal/translations/locales/es-ES/out.gotext.json

```diff
@@ -0,0 +1,245 @@
+{
+    "language": "es-ES",
+    "messages": [
```

Comment:
pls remind us why we have 2 json files in each folder.
Won't the localization process only need the en-us file for input?